packages/core/package.json

```diff
@@ -30,6 +30,7 @@
     "inversify": "^4.14.0",
     "lodash.debounce": "^4.0.8",
     "lodash.throttle": "^4.1.1",
+    "nsfw": "^1.2.0",
```

Comment:
We used to use this version of nsfw, before adopting the one from vscode. I'll check if this version is already covered by a CQ or not.

Reply:
> I'll check if this version is already covered by a CQ or not.

It's not - we had approval for `vscode-nsfw`. We'll need a new CQ.

Reply:
Do we need a CQ even for a dependency version change?

Reply:
yes for dependencies. For build and test dependencies we can request CQ with ranges ( like 1.2.0 and higher)